esptool.py

```diff
@@ -44,6 +47,8 @@ class ESPROM:
     ESP_RAM_BLOCK   = 0x1800
     ESP_FLASH_BLOCK = 0x400
 
+    ESP_FLASH_SECTOR = 0x1000
```

Comment:
Because of the commento in line 35, it's probably worth commenting here that some commands are stub-only rather than in the ROM